src/StateProvider.js

```diff
@@ -121,13 +121,19 @@ export default class StateProvider {
 
       static displayName = `ProvideState(${getDisplayName(Component)})`;
 
+      isSubscribed() {
+        return typeof this.unsubscribe === 'function';
```

Comment:
Can we change the name of this or just inline the check? this doesn't reflect whether or not the subscription is active - `this.unsubscribe()` is still a function after it's been called